app/src/main/java/org/mozilla/fenix/theme/FirefoxTheme.kt

```diff
@@ -46,9 +46,7 @@ private val darkColorPalette = FirefoxColors(
     layer3 = PhotonColors.DarkGrey30,
     layerAccent = PhotonColors.Violet40,
     layerNonOpaque = PhotonColors.Violet50A32,
-    scrim = PhotonColors.DarkGrey05A45,
-    scrimAccentStart = PhotonColors.Ink80A96,
-    scrimAccentEnd = PhotonColors.DarkGrey90A96,
+    scrim = PhotonColors.DarkGrey90A95,
```

Comment:
Layer Scrim design system colours specified on row 89 of the Changelog https://docs.google.com/spreadsheets/d/1O_LHoSqGGeXor_e2y7dpsyldm9pNJrxEbp16mD8oMCg/edit#gid=1216550615